app/views/completed_transaction/show.html.erb

```diff
@@ -47,7 +47,7 @@
 
       <form class="contact-form" action="/contact/govuk/service-feedback" method="post" id="completed-transaction-form">
         <input type="hidden" id="service_slug" name="service_feedback[slug]" value="<%= @publication.slug.gsub("done/", "") %>" />
-        <input type="hidden" id="service_done_page_url" name="service_feedback[url]" value="<%= @publication.slug %>" />
+        <input type="hidden" id="service_done_page_url" name="service_feedback[url]" value="<%= @publication.base_path %>" />
```

Comment:
Is it possible to send the full url with domain and protocol instead of just the base path?  This would avoid any future unpleasantness with munging things; particularly as it's not obvious why `slug` and `base_path` would be different, so the value for either of these inputs could be changed in the future.